pom.xml

```diff
@@ -78,7 +78,7 @@ limitations under the License.
     <!-- TODO: check if commons-codec was transitively updated to 1.13 and okhttp was updated to 2.7.5 when upgrading-->
     <beam.version>2.19.0</beam.version>
     <!-- referred from bigtable-beam-import and bigtable-emulator -->
-    <guava.version>28.2-android</guava.version>
+    <guava.version>29.0-android</guava.version>
```

Comment:
I'm a bit surprised that this needs to change

Reply:
From the original logs: https://source.cloud.google.com/results/invocations/83a30dbe-fe24-44b5-b5dc-2ea631300547/targets/cloud-devrel%2Fclient-libraries%2Fjava%2Fjava-bigtable-hbase%2Fpresubmit%2Fdependencies/log